controllers/spacebindingrequestmigration/spacebindingrequest_migration_controller.go

```diff
@@ -0,0 +1,188 @@
+package spacebindingrequestmigration
+
+import (
+	"context"
+	"fmt"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/host-operator/pkg/cluster"
+	errs "github.com/pkg/errors"
+	"github.com/redhat-cop/operator-utils/pkg/util"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+// Reconciler reconciles a SpaceBindingRequestMigration object
+type Reconciler struct {
+	Client         runtimeclient.Client
+	Scheme         *runtime.Scheme
+	Namespace      string
+	MemberClusters map[string]cluster.Cluster
+}
+
+// SetupWithManager sets up the controller with the Manager.
+func (r *Reconciler) SetupWithManager(mgr ctrl.Manager, memberClusters map[string]cluster.Cluster) error {
+	// since it's mandatory to add a primary resource when creating a new controller,
+	// we add the SpaceBindingRequest CR even if there should be no reconciles triggered from the host cluster,
+	// only from member clusters (see watches below)
+	// SpaceBindingRequest owns spacebindings so events will be triggered for those from the host cluster.
```

Comment:
The description is not accurate. We watch the SB in the host as the primary resource and SBRs in members as secondary. The actual code is correct. We just need to update the description.

Reply:
Good point. I've updated the comment as well https://github.com/codeready-toolchain/host-operator/pull/922/commits/d09374cfce39f3a9bd470735ea770348ecd347fb

Thanks!